plugins/pelican-bibtex/pelican_bibtex.py

```diff
@@ -54,9 +54,10 @@ def add_publications(generator):
     try:
         bibdata_all = Parser().parse_file(refs_file)
     except PybtexError as e:
-        logger.warn('`pelican_bibtex` failed to parse file %s: %s' % (
+        logger.error('`pelican_bibtex` failed to parse file %s: %s' % (
             refs_file,
             str(e)))
+        exit(1)
```

Comment:
I think this is an upstream package, so consider also submitting a pull request to the corresponding project.